packages/net/http/client.pony

```diff
@@ -53,6 +53,11 @@ class HTTPClient
     let valrequest: Payload val = consume request
     session(valrequest)
     valrequest
+
+  fun dispose() =>
```

Comment:
please clear the `_sessions` map as well and add a docstring describing what happens here.

Reply:
Will do.

Reply:
Done